index.js

```diff
@@ -994,29 +1042,20 @@ class GithubScm extends Scm {
      * @async  _getChangedFiles
      * @param  {Object}   config
      * @param  {String}   config.type      Can be 'pr' or 'repo'
-     * @param  {Object}   config.payload   The webhook payload received from the SCM service.
+     * @param  {Object}   [config.payload] The webhook payload received from the SCM service.
      * @param  {String}   config.token     Service token to authenticate with Github
      * @param  {String}   [config.scmUri]  The scmUri to get PR info of
      * @param  {Integer}  [config.prNum]   The PR number
      * @return {Promise}                   Resolves to an array of filenames of the changed files
      */
     async _getChangedFiles({ type, payload, token, scmUri, prNum }) {
         if (type === 'pr') {
-            let owner;
-            let repo;
-            let number;
-
-            if (scmUri && prNum) {
-                const scmInfo = await this.lookupScmUri({ scmUri, token });
-
-                owner = scmInfo.owner;
-                repo = scmInfo.repo;
-                number = prNum;
-            } else {
-                owner = hoek.reach(payload, 'repository.owner.login');
-                repo = hoek.reach(payload, 'repository.name');
-                number = hoek.reach(payload, 'number');
-            }
+            await this.getPrMergeable({ scmUri, token, prNum }, 0);
```

Comment:
Should we need to be included this in `try` ?

Reply:
I fixed it.